packages/core/src/test/awsService/cloudWatchLogs/registry/liveTailSession.test.ts

```diff
@@ -0,0 +1,97 @@
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import assert from 'assert'
+import { LiveTailSession } from '../../../../awsService/cloudWatchLogs/registry/liveTailSession'
+import { StartLiveTailCommand } from '@aws-sdk/client-cloudwatch-logs'
+describe('LiveTailSession', async function () {
+    const testLogGroupArn = 'arn:aws:test-log-group'
+    const testRegion = 'test-region'
+    const testFilter = 'test-filter'
+    const testAwsCredentials = {} as any as AWS.Credentials
+    it('builds StartLiveTailCommand: no stream Filter, no event filter.', function () {
+        const session = new LiveTailSession({
+            logGroupArn: testLogGroupArn,
+            region: testRegion,
+            awsCredentials: testAwsCredentials,
+        })
+        assert.strictEqual(
+            JSON.stringify(session.buildStartLiveTailCommand()),
+            JSON.stringify(
+                new StartLiveTailCommand({
+                    logGroupIdentifiers: [testLogGroupArn],
+                    logEventFilterPattern: undefined,
+                    logStreamNamePrefixes: undefined,
+                    logStreamNames: undefined,
+                })
+            )
+        )
+    })
+    it('builds StartLiveTailCommand: with prefix stream Filter', function () {
+        const session = new LiveTailSession({
+            logGroupArn: testLogGroupArn,
+            logStreamFilter: {
+                type: 'prefix',
+                filter: 'test-filter',
+            },
+            region: testRegion,
+            awsCredentials: testAwsCredentials,
+        })
+        assert.strictEqual(
+            JSON.stringify(session.buildStartLiveTailCommand()),
+            JSON.stringify(
+                new StartLiveTailCommand({
+                    logGroupIdentifiers: [testLogGroupArn],
+                    logEventFilterPattern: undefined,
+                    logStreamNamePrefixes: [testFilter],
+                    logStreamNames: undefined,
+                })
+            )
+        )
+    })
+    it('builds StartLiveTailCommand: with specific stream Filter', function () {
+        const session = new LiveTailSession({
+            logGroupArn: testLogGroupArn,
+            logStreamFilter: {
+                type: 'specific',
+                filter: 'test-filter',
+            },
+            region: testRegion,
+            awsCredentials: testAwsCredentials,
+        })
+        assert.strictEqual(
+            JSON.stringify(session.buildStartLiveTailCommand()),
+            JSON.stringify(
+                new StartLiveTailCommand({
+                    logGroupIdentifiers: [testLogGroupArn],
+                    logEventFilterPattern: undefined,
+                    logStreamNamePrefixes: undefined,
+                    logStreamNames: [testFilter],
+                })
+            )
+        )
+    })
+    it('builds StartLiveTailCommand: with log event Filter', function () {
```

Comment:
most of the tests look nearly identical. the copy-paste detection CI will flag this, and is worth fixing.